libraries/core_libs/network/src/tarcap/packets_handlers/dag_block_packet_handler.cpp

```diff
@@ -112,43 +90,60 @@ void DagBlockPacketHandler::sendBlock(dev::p2p::NodeID const &peer_id, taraxa::D
 }
 
 void DagBlockPacketHandler::onNewBlockReceived(DagBlock &&block, const std::shared_ptr<TaraxaPeer> &peer) {
-  if (dag_blk_mgr_) [[likely]] {
+  if (dag_mgr_) [[likely]] {
```

Comment:
does this need to be separate function? process is doing almost nothing and if we are not calling this from somewhere else I would remove it and move this code to process function

Reply:
We call it from one old test, maybe this test should be refactored

Reply:
TODO: Fix in https://github.com/Taraxa-project/taraxa-node/issues/1895